tools/distrib/check_copyright.py

```diff
@@ -173,7 +214,12 @@ def log(cond, why, filename):
     if m:
         pass
     elif 'DO NOT EDIT' not in text:
-        log(1, 'copyright missing', filename)
+        if args.fix:
+            text = license_text + '\n\n' + text
+            open(filename, 'w').write(text)
+            log(1, 'copyright missing (fixed)', filename)
+        else:
+            log(1, 'copyright missing', filename)
         ok = False
```

Comment:
Maybe you should advertise this new functionality if it failed. E.g., you can add following to line 224:

```py
if not ok:
    print('You may use following command to automatically fix copyright headers:')
    print('    tools/distrib/check_copyright.py --fix')
```

Reply:
That's a good idea, thanks!